_posts/2017-07-07-fastlane-ios.md

```diff
@@ -0,0 +1,703 @@
+---
+layout: post
+title: Fastlane, vers l'infini et l'au dela
+excerpt: Nous allons découvrir un outil qui permet d'automatiser des tâches fastidieuses en mobile.
+permalink: /fr/fastlane/
+categories:
+    - Ios
+    - Swift
+    - Xcode
+tags:
+    - Ios
+    - Swift
+    - Xcode
+    - Tutorial
+image:
+  path: /assets/2017-07-17-fastlane-ios/fastlane_logo.png
+  height: 100
+  width: 100
+---
+
+# Qu'est-ce que "Fastlane"
+
+Fastlane est un outil open-source qui permet de faire du **Continuous Delivery** sous ios et android.
+Il permet d'automatiser un certain nombre de tâches fastidieuses comme gèrer les sreenshots, les certificats, deployer votre app ...
+
+Vous l'avez compris, cet outil va changer votre vie.
+
+Voici la liste des libs à dispositions avec Fastlane :
+
+<img src="../../assets/2017-07-17-fastlane-ios/fastlane_tools.png" width="400" alt="Fastlane tools">
```

Comment:
markedown instead of html: `![](/assets/{post-title}/{asset-name-png-or-jpg) Title`